README.md

```diff
@@ -72,15 +72,16 @@ It also has several useful helpers to handle errors.
 
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
-| auth | `Object <username, password, targetBranch>` | - | See `auth` description [below](#auth). |
+| auth | `Object <username, password, targetBranch, pcc, overridePcc>` | - | See `auth` description [below](#auth). |
```

Comment:
If it's used only for search results, I think we have to remove it, as any user of library takes control of what's going on.